docs/news.rst

```diff
@@ -51,6 +48,12 @@ Backward-incompatible changes
     instead of being defined as a generator, is now executed *after* the
     :ref:`scheduler <topics-scheduler>` instance has been created.
 
+-   When using :setting:`JOBDIR`, :ref:`start requests <start-requests>` are
+    now serialized into their own, ``s``-suffixed priority folders.
+
+    Set :setting:`SCHEDULER_START_DISK_QUEUE` to ``None`` or ``""`` to change
+    that.
```

Comment:
It'd be good to document what happens when the option is set to None - it currently reads as it chages the behavior mentioned previously (separate folders, etc), but it doesn't tell what's the behavior after the chage.